<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TUPperware Save Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/TUPperware_logo.png</normaloff>img/TUPperware_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="input_path_line_edit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Single Click: Open file dialog
Double Click: Type in the text box </string>
    </property>
    <property name="placeholderText">
     <string>Path to your TLGameSave / TLGameSettings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_file_push_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Load the save file from the path in the text box</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="log_text_browser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>480</width>
      <height>100</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Information and logs will be printed here</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="save_tree_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>480</width>
      <height>200</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::EditKeyPressed</set>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="headerCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>210</number>
    </attribute>
    <column>
     <property name="text">
      <string>Key</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="output_path_line_edit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Single Click: Open file dialog
Double Click: Type in the text box </string>
    </property>
    <property name="placeholderText">
     <string>Path to the output save file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_push_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Export the save file to the path in the text box</string>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cipher_output_check_box">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Ciphered Data</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
